features/ui/button/button.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import {
+  ButtonContainer,
+  ButtonSize,
+  ButtonColor,
+  ButtonIcon,
+  ButtonContainerProps,
+} from "./button";
+
+export default {
+  title: "UI/ButtonContainer",
+  component: ButtonContainer,
+  parameters: {
+    layout: "fullscreen",
+  },
+} as ComponentMeta<typeof ButtonContainer>;
+
+const Template: ComponentStory<typeof ButtonContainer> = ({
```

Comment:
Great job on creating your first story!

Reply:
Btw if you move the default props to the `Default.args` object like this

```
const Template: ComponentStory<typeof ButtonContainer> = ({
  children,
  size,
  color,
  icon,
  disabled,
}: ButtonContainerProps) => (
  <div style={{ padding: 50 }}>
    <ButtonContainer size={size} color={color} icon={icon} disabled={disabled}>
      {children}
    </ButtonContainer>
  </div>
);

export const Default = Template.bind({});
Default.args = {
  children: "Button CTA",
  size: ButtonSize.md,
  color: ButtonColor.primary,
  icon: ButtonIcon.none,
  disabled: false,
};
```

you get more controls in Storybook (currently you have two) and thus can test more states

![image](https://user-images.githubusercontent.com/9722792/189607461-68d9043a-8d7b-4ab5-bf28-db8ffe4fce81.png)